assets/scripts/dialogs/Geotag/GeoSearch.jsx

```diff
@@ -3,13 +3,13 @@ import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import DownshiftPelias from 'downshift-pelias'
 import Pelias from 'pelias-js'
-import { injectIntl, intlShape } from 'react-intl'
+import { injectIntl } from 'react-intl'
 import { PELIAS_HOST_NAME, PELIAS_API_KEY } from '../../app/config'
 import { setMapState } from '../../store/actions/map'
 
 class GeoSearch extends React.Component {
   static propTypes = {
-    intl: intlShape.isRequired,
+    intl: PropTypes.object.isRequired,
```

Comment:
Not a huge deal, but I it looks like in v3 react-intl exports an `[IntlShape](https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#typescript-support)`, could we use that here with `PropTypes.shape`?

Reply:
Didn't work, and I suppose that's to be expected — I don't think TypeScript and PropTypes are compatible, as they're totally different systems for typechecking.